frontend/packages/shared/src/navigation/main-header/profileMenu.tsx

```diff
@@ -26,13 +27,18 @@ export function ProfileMenu({ showlogout, user }: IProfileMenuComponentProps) {
 
   return (
     <div>
-      <img
-        src={user.avatar_url}
-        className={classes.userAvatar}
-        aria-haspopup
-        onClick={handleClick}
-        aria-label='profilikon'
-      />
+      {window.location.pathname.includes('preview') ? (
+        <PersonCircleIcon className={classes.previewProfilIcon} title='profile Icon' />
```

Comment:
I actually think this change applies for the general header design, and not just for preview. So you could remove the preview-check and remove the `img` block below, and just keep the `<PersonCircleIcon...` instead 😊

Reply:
Done